src/software/thunderscope/thunderscope_config.py

```diff
@@ -640,7 +649,10 @@ def get_extra_widgets(proto_unix_io):
                     else [
                         configure_robot_view_diagnostics(
                             proto_unix_io_map[ProtoUnixIOTypes.DIAGNOSTICS]
-                        )
+                        ),
+                        configure_estop(
+                            proto_unix_io_map[ProtoUnixIOTypes.DIAGNOSTICS]
+                        ),
                     ],
```

Comment:
This logic only adds these widgets if we're only running diagnostics (no fullsystem). Ideally we should show these widgets in diagnostics anyways just to reduce the confusion of having to jump back and forth between the tabs.

From quick testing, it seems like with the current implementation of robot view widget, the state of the control mode is local to the widget (we subscribe to `control_mode_menu` in `thunderscope_main.py` for only 1 of the robot view widgets?!). Not sure if Qt allows us to initialize the widget once but visualize it in both tabs so we don't have to handle the state consistency logic. If that's not easily possible, we can ignore adding robot view widget to diagnostics tab for now.
But the EStop tab should be added by default, and the state for that works across multiple estop widgets as it reads the value from the USB/Keyboard.